app/components/certificate/description.tsx

```diff
@@ -57,7 +59,24 @@ export default function DescriptionSection({
             <Flex justifyContent="flex-end">
               <Form method="post">
                 <Button type="submit" rightIcon={<RepeatIcon />} isDisabled={!isRenewable}>
-                  Renew Certificate
+                  Renew
+                </Button>
+              </Form>
+              <Form method="post">
+                <input type="hidden" name="intent" value="delete-certificate" />
+                <Button
```

Comment:
Let's use `variant="outline"` or `variant="ghost"` to make this less prominent.

Reply:
If the goal is to make it less prominent, should we use `IconButton` and `Tooltip`?

Reply:
I think having it be a button is good, just that we only have a single "Call to Action" button (i.e., refresh).